tests/unit/frameworks/pytorch/restore_utils_test.py

```diff
@@ -193,15 +199,17 @@ def test_load_nonlinear(self):
             tot_eq = (param1 == param2).sum().item()
             self.assertNotEqual(tot_eq, np.prod(param1.shape))
 
-        # Check output through the lower network.
-        out = lower_forward(model, self.in_1)
-        num_matches = out.isclose(self.out_lower, atol=1e-2).sum().item()
-        self.assertEqual(num_matches, 1337)  # some correct
-
-        # Check output through the lower network.
-        out = upper_forward(model, self.in_2)
-        num_matches = out.isclose(self.out_upper, atol=1e-2).sum().item()
-        self.assertEqual(num_matches, 1)  # some correct
+        # These exact numbers are random.
```

Comment:
Instead of commenting out the test could you add a "skip" comment just like you added to `test_checkpoint_backward_compatibility`?